pages/common/ffmpeg.md

```diff
@@ -25,3 +25,7 @@
 - Remux MKV video to MP4 without re-encoding audio or video streams:
 
 `ffmpeg -i {{input_video}}.mkv -codec copy {{output_video}}.mp4`
+
+- Convert MP4 video to VP9 codec. The CRF value can be from 0–63. Lower values mean better quality. -b:v MUST be 0:
```

Comment:
Rewording this to something like "For the best quality, use a CRF value (recommended range 15-35) and -b:video MUST be 0:"

Admittedly it is a pain getting what is actually reasonably technical and nuanced information to a user who just wants something that will work.